bots/notify/src/main/java/org/openjdk/skara/bots/notify/prbranch/PullRequestBranchNotifier.java

```diff
@@ -69,13 +65,17 @@ private void deleteBranch(PullRequest pr) {
         if (protectBranches) {
             log.info("Removing branch protection for " + branch);
             pr.repository().unprotectBranchPattern(branch);
+            log.info("Removing branch protection for *");
+            pr.repository().unprotectBranchPattern("*");
         }
         if (!branchExists) {
             log.info("Pull request pre-integration branch " + branch + " doesn't exist on remote - ignoring");
             return;
         }
         log.info("Deleting pull request pre-integration branch " + branch);
         pr.repository().deleteBranch(branch);
+        log.info("Protecting branch * after deleting branch " + branch);
+        pr.repository().protectBranchPattern("*");
```

Comment:
We should not be touching branch protection settings unless `protectBranches` is enabled.

Reply:
Exactly, will fix it.